### PR TITLE
anagram-exercise: Added anagram exercise.

### PR DESCRIPTION
**Pull request by SimaDovakin**
An exercise for #48in24. I used test cases from the C implementation.

### CODE REVIEW

File: exercises/practice/anagram/.docs/instructions.append.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Hints
```
Comment (ErikSchierboom) [inline exercises/practice/anagram/.docs/instructions.append.md:1]:
```suggestion
# Instructions append
```

Comment (SimaDovakin) [reply exercises/practice/anagram/.docs/instructions.append.md:1]: Sure. I'll change

**Review by ErikSchierboom — APPROVED** [review]

**Comment by vendethiel** [comment]
The fetch-cobolcheck{,.ps1} files are empty, is that on purpose?

**Comment by SimaDovakin** [comment]
Ohh, I forgot about them. I'll fix it

**Comment by SimaDovakin** [comment]
Fix for fetch-cobolcheck{,.ps1} files: https://github.com/exercism/cobol/pull/156

**Comment by vendethiel** [comment]
Thanks

**Pull request merged**